docs/decisions/0004-which-cookiecutters-included.rst

```diff
@@ -0,0 +1,33 @@
+4. Which Cookiecutters in edx-cookiecutters?
+============================================
+
+Status
+------
+
+Accepted
+
+
+Context
+-------
+
+* Currently, the only cookiecutters in this repository are the Open edx public python cookiecutters.
+
+* Since the creation of this repo, others have tried adding public non-python cookiecutters to this repository.
+
+* The name of this repository suggests all public edx cookiecutters will be in this repository.
+
+* We want to make it easier to find a cookiecutter and create a new project. This would encourage community contribution.
+
+
+Decision
+--------
+
+edx-cookiecutter should be the central point for public edx cookiecutters. Most Open edx public cookiecutters should be placed in this repository.
+
+If there is a complelling case why a cookiecutter should be elsewhere, a link to it should be added the edx-cookiecutter/README.rst.
+
+
+Consequences
+------------
+
+All edx cookiecutters should be moved to this repository.
```

Comment:
I think it's worth mentioning separating front-end and backend cookiecutters but that we chose to not do that because there might be future projects that build upon both.